cocos/tiledmap/tmx-xml-parser.ts

```diff
@@ -89,7 +89,7 @@ function strToColor (value: string): Color {
 
 function getPropertyList (node: Element, map?: PropertiesInfo): PropertiesInfo {
     const res: any[] = [];
-    const properties = node.getElementsByTagName('properties');
+    const properties = node.querySelectorAll(':scope > properties');
```

Comment:
WeChat mini-games do not support this interface

Reply:
@Jerry457

Reply:
fixed

Reply:
@Jerry457 Please fix test error.

Reply:
And modify the branch to 3.8.5

Reply:
Fixed in new PR: https://github.com/cocos/cocos-engine/pull/17641